### ISSUE TITLE
multiple websocket connections (client)

### ISSUE THREAD

**Issue by fsteff** [RangerMauve/discovery-swarm-stream#6]
It would be useful to support multiple websocket connections with just one instance of the proxy.
E.g. imagine there were multiple gateway servers plus a local one. Each may provide different peers, so it would be useful to connect to all of them.

**Comment by RangerMauve** [comment]
Yeah, and having auto-reconnect logic to cycle between the servers.
